package.json

```diff
@@ -41,10 +41,18 @@
       "type": "object",
       "title": "stylefmt configuration",
       "properties": {
-        "stylefmt.useStylelintConfigOverrides": {
```

Comment:
I think we should revert this changes, because this feature is related to issue #8.